index.html

```diff
@@ -19,6 +19,18 @@
             <img class="social_media_icons" src="icons/Union.png" alt="">
         </nav>
     </header>
+    <!--Desktop View-->
+    <header class="desktop_header">
+        <!--header section-->
+        <nav class="desktop_navigation_bar">
+            <p><span>My Logo</span></p>
```

Comment:
- [ ][**OPTIONAL**] The logo should be wrap inside a `a` tag to allow use to navigate freely within the webpage , kindly change that ,it will increase the user experience :+1: